src/main/java/meilisearch/IndexesHandler.java

```diff
@@ -0,0 +1,98 @@
+package meilisearch;
+
+import com.google.gson.JsonObject;
+
+/**
+ * Wrapper around the MeilisearchHttpRequest class to ease usage for Meilisearch indexes
+ */
+class IndexesHandler {
```

Comment:
This new class name makes it easier to understand what this class is actually does